### PR TITLE
Add AZ info to node detail

### PR DESCRIPTION
**Pull request by amegianeg**

### CODE REVIEW

**Comment by fermayo** [comment]
there is also an AZ section to fill in

**Comment by amegianeg** [comment]
Updated

File: source/includes/availabilityzone.md [hunk 1]
```diff
@@ -1,5 +1,102 @@
...
+--------- | -----------
+name | Filter by availability zone name
+region | Filter by region (resource URI)
+
+
+
+## Get an individual availability zone
+
+```python
+# TODO
```
Comment (fermayo) [inline source/includes/availabilityzone.md]: remove this TODO

File: source/includes/availabilityzone.md [hunk 2]
```diff
@@ -1,5 +1,102 @@
...
+```
+
+```go
+# TODO
```
Comment (fermayo) [inline source/includes/availabilityzone.md]: remove this TODO

File: source/includes/availabilityzone.md [hunk 3]
```diff
@@ -1,5 +1,102 @@
...
\ No newline at end of file
+## Availability Zone
+
+> Example
+
+```json
+{
+    "available": true,
+    "name": "ap-northeast-1a",
+    "region": "/api/v1/region/ap-northeast-1/",
```
Comment (fermayo) [inline source/includes/availabilityzone.md]: this is not true, should be `/api/v1/region/aws/ap-northeast-1/`

**Comment by fermayo** [comment]
@amegianeg reopen against staging

**Pull request closed without merging**